### DISCUSSION TITLE
Tabbing between inputs not working

### DISCUSSION THREAD

**Discussion by sebm** [twilio-labs/paste#774] [Q&A] [answered]
I have a page configured with several `<FormInput>` components that I can navigate between using the Tab key. This page has a modal that it loads, with a few `<FormInput>`s on it, and i cannot tab between them. As far as I can tell the implementations are very similar, so I'm not sure what's up. I've tried rejigging the JSX elements, settings `tabIndex`, but no luck.

I don't want to post the details on public GH so please hit me up on Slack if you're able to help 👍

**Comment by SiTaggart** [comment, marked as answer]
HI @sebm 

Would it be possible for you to create a minimal reproduction in Codesandbox please? We have [examples on the doc site that have inputs inside of modals](https://paste.twilio.design/components/modal#wide-modal) that seem to work just fine for me, so a reproduction in a clean environment is the only way I can help debug your issue right now.

Thanks

**Reply by sebm** [reply]
As far as I can tell the problem is with the deprecated internal FullscreenModal component, which seems to be stealing Tab events somehow… is there an equivalent Paste component?

**Reply by serifluous** [reply]
@sebm sorry for the late follow-up! We don't have an equivalent full-screen modal in Paste yet. Since there are just a few form inputs you're putting in the Modal, would you be able to swap to a default-sized modal?

If not, feel free to drop a screenshot of what you have in the #help-design-system channel, and we could help give some design recommendations.

**Reply by sebm** [reply]
hey @serifluous thanks for chiming in – alas i think a full screen modal is necessary, although there are only a few form inputs there's a Table element that occupies a reasonable amount of space. Is there a timeline for implementing a fullscreen modal in paste?

**Reply by serifluous** [reply]
Yep, makes sense. We're in the middle of talking to a few BUs to re-prioritize our component roadmap, but so far, full-screen modals are planned for a late January release.

**Reply by serifluous** [reply]
Also, I'm assuming this is for V&V—made a note for us that you'll need it! Let me know if I got the BU wrong.

**Reply by sebm** [reply]
Thanks @serifluous 
Funnily enough, it's actually for risk platform 😂
